jobManager.py

```diff
@@ -108,7 +107,8 @@ def __manage(self):
                     "%s|Dispatched job %s:%d [try %d]"
                     % (datetime.utcnow().ctime(), job.name, job.id, job.retries)
                 )
-
+                # Mark the job assigned
+                self.jobQueue.assignJob(job.id, preVM)
```

Comment:
```
if preVM is None:
    job.makeAssigned()
else:
    self.jobQueue.assignJob(job.id, preVM)
```
makes this work locally for me... still can't get autolab-docker to work so I haven't been able to test it there though

Reply:
ah but this makes it work on docker lol, which is so confusing because they essentially run the same code but in different places...

Reply:
Actually nvm I haven't tried this with the edited comment. Let me see what happens